plugins/teiid-8.11/src/main/java/org/komodo/teiid/TeiidInstanceImpl.java

```diff
@@ -253,6 +254,16 @@ public TeiidTranslator getTranslator(String name) throws Exception {
         return teiidTranslators;
     }
 
+    private boolean isDynamic(VDB vdb) {
```

Comment:
This identical code block is written at least 3 times in different classes. Is there a way to make it a static method somewhere?